packer/instance-scripts/init-bootnode.sh

```diff
@@ -20,25 +20,32 @@ function complete_constellation_config {
 
     local REGIONS=$(cat /opt/quorum/info/regions.txt)
 
-    # Configure constellation with other node IPs
-    OTHER_NODES=""
-    for region in ${REGIONS[@]}
-    do
-        local NUM_BOOTNODES=$(cat /opt/quorum/info/bootnode-counts/${region}.txt)
-        for index in $(seq 0 $(expr $NUM_BOOTNODES - 1))
+    DIR=$(dirname "${CONSTELLATION_CONFIG_PATH}")
```

Comment:
Make these variables local please. If you do need access to it outside the function, create the variable in the main code path and pass it to the function as an argument.

Reply:
After making these changes, do I do a new PR, or what? No idea how to proceed.

Reply:
See main thread comment below.